README.md

```diff
@@ -4,6 +4,50 @@ A module for FeedHenry RainCatcher that manages users, groups and memberships. I
 - Backend services to handle CRUD operations for user, group and membership.
 - Frontend directives and services providing CRUD clients for user, group and membership.
 
+## Upgrading to 0.2.0 from 0.1.x
+Version 0.2.0 introduces session storage for authenticated users utilizing either mongodb or redis as backends.
+
+This involves an extra parameter to the initialization of the router for the authentication service which contains the configuration for the session storage.
+
+### How to upgrade
+
+In the auth-service:
```

Comment:
Maybe: In the MBaaS service that authenticates users (e.g. raincatcher-demo-auth), initialise the session store with the configuration shown below.